ca/certificate-authority.go

```diff
@@ -599,9 +603,16 @@ func (ca *CertificateAuthorityImpl) IssueCertificate(csr x509.CertificateRequest
 	}
 
 	// Submit the certificate to any configured CT logs
-	go func() {
-		_ = ca.Publisher.SubmitToCT(certDER)
-	}()
+	if ca.GRPCPublisher != nil {
+		ctx, _ := context.WithTimeout(context.Background(), ca.GRPCTimeout)
```

Comment:
As this is starting a new context, the variable should be named something other than `ctx` for when the function itself gets a context.